src/test/java/uk/gov/ons/ctp/integration/contactcentresvc/service/impl/CaseServiceImplFulfilmentTest.java

```diff
@@ -56,17 +56,11 @@ public void setup() {
   public void testFulfilmentRequestByPost_individualFailsWithNullContactDetails() throws Exception {
     // All of the following fail validation because one of the contact detail fields is always null
     // or empty
-    doVerifyFulfilmentRequestByPostFailsValidation(
```

Comment:
I think we could do with a unit tests that succeeds for fulfilment by POST for individual where title is null or blank.